tests/smoke/block_storage_smoke_test.go

```diff
@@ -100,12 +104,12 @@ func (suite *SmokeSuite) TearDownSuite() {
 // When creating pvc in k8s platform, it may take some time for the block Image to be bounded
 func retryBlockImageCountCheck(h *SmokeTestHelper, imageCount int, expectedChange int) bool {
 	inc := 0
-	for inc < 30 {
+	for inc < 60 {
 		blockImages, _ := h.GetBlockClient().BlockList()
 		if imageCount+expectedChange == len(blockImages) {
 			return true
 		} else {
-			time.Sleep(time.Second * 2)
+			time.Sleep(time.Second * 3)
```

Comment:
again, would be good to have error checks so we don't wait for the timeout in error conditions